deployment/node-feature-discovery/values.yaml

```diff
@@ -0,0 +1,195 @@
+image:
+  repository: gcr.io/k8s-staging-nfd/node-feature-discovery
+  # This should be set to 'IfNotPresent' for released version
+  pullPolicy: Always
+imagePullSecrets: []
+  
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+nameOverride: ""
+fullnameOverride: ""
+
+master:
```

Comment:
this is defaulting `nfd-master` as a replicaset and not daemonset, why?

Reply:
I'm not sure that I understood you right. nfd-master is a Deployment: https://github.com/kubernetes-sigs/node-feature-discovery/pull/423/files#diff-a4f5c7e13b9887fcc6c41a624fa5b34e46042d4b1dcaf2401c78e98c7451ccedR2

Reply:
Is ok, I am super used to run NFD as a daemon set, but this is ok and works just fine